### ISSUE TITLE
check repo for secrets in git history

### ISSUE THREAD

**Issue by ExperimentsInHonesty** [hackforla/tables#57]
### Overview
We need to remove any secrets from the code base so that we can make the repo public again.

### Action Items
- [ ] Find app to search for secrets in repo, document below
- [x] Check history to find any secrets and document them below
- [x] Define any secrets we need to update.

### Resources/Instructions
- [Search string to look for apps](https://www.google.com/search?q=search+github+repo+for+secrets&oq=search+github+repo+for+secrets&gs_lcrp=EgZjaHJvbWUyBggAEEUYOTIICAEQABgWGB4yCggCEAAYhgMYigUyCggDEAAYhgMYigUyCggEEAAYhgMYigUyCggFEAAYhgMYigXSAQkxNDc3MmowajeoAgCwAgA&sourceid=chrome&ie=UTF-8)
- [gitleaks](https://github.com/gitleaks/gitleaks)

**Comment by Reltre** [comment]
I used the recommended tool for looking for potential secrets in the [tables](https://github.com/hackforla/tables/tree/main) git history. 

<img width="242" alt="Screenshot 2024-09-18 at 2 18 39 PM" src="https://github.com/user-attachments/assets/b612daf8-c2fd-48fb-9ba9-fad521a3c5c7">

I used this tool on the main branch and all existing branches on the repo. I also did a manual scan of the commit history.
I didn't notice any secrets in the git history. Mainly just various links to tools we might use like scripts and assets.
